eoc_journal/templates/eoc_journal.html

```diff
@@ -4,30 +4,108 @@
     <h3>{{ display_name }}</h3>
   </div>
 
-  {% if answer_sections %}
-  <div class="eoc-selected-answers">
-    {% for section in answer_sections %}
-      <section class="eoc-selected-answers-section">
-        <h2>{{ section.name }}</h2>
-        {% for q in section.questions %}
-          <div class="eoc-selected-answer">
-            <h3>{{ q.question }}</h3>
-            <p>{{ q.answer }}</p>
-          </div>
-        {% endfor %}
-      </section>
-      {% endfor %}
+  <div class="progress-metrics">
+    <div class="title">
+      <h4>{% trans "Progress" %}</h4>
+      <p>
+
+      {% if progress %}
+
+      {% blocktrans with percentage=progress.user tag_start='<span data-progress-name="user"></span>' tag_end='</span>' %}
+      You are {{ tag_start }}{{ percentage }}{{ tag_end }}% of the way through
+      the course based on completion of modules within lessons.
+      {% endblocktrans %}
+
+      {% blocktrans with percentage=progress.cohort_average tag_start='<span data-progress-name="cohort"></span>' tag_end='</span>' %}
+      The cohort average is {{ tag_start }}{{ percentage }}{{ tag_end }}.
+      {% endblocktrans %}
+
+      {% else %}
+      {% trans "Progress data is not available." %}
+      {% endif %}
+      </p>
+    </div>
   </div>
-  {% endif %}
-
-  <p>
-    {% if key_takeaways_pdf_url %}
-    <a class="key-takeaways-link" target="_blank" href="{{ key_takeaways_pdf_url }}">
-      <span class="fa fa-file-text-o" aria-hidden="true"></span>
-      {% trans "Key Takeaways" %}
-    </a>
-    {% else %}
-      {% trans "Key Takeaways PDF not available at this time." %}
-    {% endif %}
-  </p>
+
+  <div class="engagement-metrics">
+    <div class="title">
+      <h4>{% trans "Engagement" %}</h4>
+
+      <p>
+      {% if engagement %}
+
+      {% trans "Your current engagement score is "%} <span data-engagement-name="user">{{engagement.user_score}}</span>
+      {% trans "based on your engagement interactions with other participants in your cohort in the discussion" %}
+      {% trans "forums. The cohort average is" %} <span data-engagement-name="cohort">{{engagement.cohort_score}}</span>
```

Comment:
This should use two `blocktrans` blocks with HTML tags being interpolated via `{{tag_start}}` and `{{tag_end}}`, just like several lines above.

Also, sentences should not be split between multiple `trans` or `blocktrans`.

In this case I think we have to use two `blocktrans` blocks:

1. "Your current engagement score is {{tag_start}}{{score}}{{tag_end}} based on your engagement interactions with other participants in your cohort in the discussion forums."
2. "The cohort average is {{tag_start}}{{score}}{{tag_end}}."